src/docfx/lib/cache/JsonDiskCache.cs

```diff
@@ -133,6 +127,19 @@ public async Task<TError[]> Save()
             });
         }
 
+        private void UpdateCache(TValue value)
+        {
+            foreach (var cacheKey in value.GetKeys())
+            {
+                if (!_cache.TryGetValue(cacheKey, out var existingValue) || HasExpired(existingValue))
```

Comment:
>_cache.TryGetValue(cacheKey, out var existingValue) [](start = 21, length = 51)

It won't hit here when not expired, right?

Reply:
it can, if value exists or expired(not and)